src/main/java/org/fundacionjala/coding/cynthia/Video/Customer.java

```diff
@@ -0,0 +1,83 @@
+package org.fundacionjala.coding.cynthia.Video;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * costumer class.
+ */
+public class Customer {
+    private String customerName;
+    private List<Rental> movieRentals;
+    private double totalAmount = 0;
+    private int frequentRenterPoints;
```

Comment:
I think these attributes are no longer necessary because you have methods that are returned the totalAmount and totalFrequentRenterPoints.